README.md

```diff
@@ -779,6 +779,7 @@ Table of Contents
   * [jare.io](http://www.jare.io) — CDN for images. Uses AWS CloudFront
   * [jsdelivr.com](https://www.jsdelivr.com/) — CDN of OSS (JS, CSS, fonts) for developers and webmasters, accepts PRs to add more
   * [Microsoft Ajax](https://docs.microsoft.com/en-us/aspnet/ajax/cdn/overview) — The Microsoft Ajax CDN hosts popular third-party JavaScript libraries such as jQuery and enables you to easily add them to your Web application
+  * [MerlinCDN](https://www.merlincdn.com/) - CDN product that offers Unlimited number of Requests, Automated Free SSL, Visitor and Traffic Statistics, User and Source Izolation for different projects of webmaster and/or company, Last 1000 Logs of the traffic, Trial Credit for new users.
```

Comment:
Whats the details of the free offering? If it's only a trial it would not be applicable to this list

Reply:
Hey @ripienaar 
Do you remember that they opened another such PR recently ?
They just wanna add this on the list although they're offering only a trial.
They think that this will help them advertise for free I guess.

Reply:
@thispsj  I remember, I am asking a bit toungue in cheek hoping they'd think for themselves and realise this is a list not for free trials :)

Reply:
Signed up to their service and they seem to offer a 10$ recurring credits everymonth, basically like Railway,Qovery and stuff.
Why cannot it be added to the list?
p.s I am not affiliated with Merlin, or have connections with them

Reply:
Great if thats the case, but its not something I can confirm without signing up, I am not  signing up for 1000s of services to verify things like this.  If the sites not clear it either clarifies or doesnt get listed.

Given the lack of response here I'll close this, if @merlincdn wants to clarify whats free feel free to reopen.

Reply:
We suppose that free 10 dollars is also included in the limitations of the freefordevelopers repo since there are some other items in the list offering a limited amount of money for registration. So that would be great if you can consider these points again.

Reply:
@merlincdn if you can point those out we will remove them.